File: berkeley/ruby/delete/berkeley_delete.rb
#!/usr/bin/ruby
# -*- coding: utf-8 -*-
#
#	berkeley_delete.rb
#
#					Mar/07/2014
#    
require 'bdb'
require	'json'
#
load '/var/www/data_base/common/ruby_common/kvalue_manipulate.rb'
# --------------------------------------------------------------------
puts "*** 開始 ***"
db_file = ARGV[0]
id_in = ARGV[1]

puts id_in
#
ee = BDB::Btree::open(db_file,"table1","r+",0777)
kvalue_delete_proc(ee,id_in)
ee.close()
#
puts "*** 終了 ***"

File: berkeley/python/delete/berkeley_delete.py
#! /usr/bin/python3
# -*- coding: utf-8 -*-
#
#	berkeley_delete.py
#
#					Jan/06/2015
import	sys
from bsddb3 import db
import string
#
sys.path.append ('/var/www/data_base/common/python_common')
from dbm_manipulate import dbm_delete_proc
# -------------------------------------------------------------
print ("*** 開始 ***")
#
db_name = sys.argv[1]
key_in = sys.argv[2]
print ("%s" % db_name)
print ("%s" % key_in)
#
adb = db.DB ()
adb.open (db_name)
dbm_delete_proc	(adb,key_in)
#
adb.close ()
#
print	("*** 終了 ***")
# -------------------------------------------------------------
